<ui version="4.0" >
 <class>ViewMPEG2</class>
 <widget class="QWidget" name="ViewMPEG2" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>319</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>k9Copy - Title Preview</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="gbCrop" >
     <property name="title" >
      <string>Crop</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Top</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="sbTop" >
        <property name="maximum" >
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="sbLeft" >
        <property name="maximum" >
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Bottom</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="sbBottom" >
        <property name="maximum" >
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="sbRight" >
        <property name="maximum" >
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1" >
       <spacer name="verticalSpacer" >
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1" >
       <widget class="QPushButton" name="bCropDetect" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>detect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QFrame" name="frame7" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>32767</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QToolButton" name="bStop" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset>
          <normaloff>image1</normaloff>image1</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QToolButton" name="bPlay" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset>
          <normaloff>image2</normaloff>image2</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QSlider" name="slider" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>bStop</sender>
   <signal>clicked()</signal>
   <receiver>ViewMPEG2</receiver>
   <slot>bStopClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>38</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bPlay</sender>
   <signal>clicked()</signal>
   <receiver>ViewMPEG2</receiver>
   <slot>bPlayClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>66</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider</sender>
   <signal>sliderReleased()</signal>
   <receiver>ViewMPEG2</receiver>
   <slot>sliderReleased()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>94</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider</sender>
   <signal>sliderPressed()</signal>
   <receiver>ViewMPEG2</receiver>
   <slot>sliderPressed()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>94</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bCropDetect</sender>
   <signal>clicked()</signal>
   <receiver>ViewMPEG2</receiver>
   <slot>bCropDetectClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>426</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel" >
     <x>477</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbTop</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ViewMPEG2</receiver>
   <slot>sbValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>454</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel" >
     <x>482</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbLeft</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ViewMPEG2</receiver>
   <slot>sbValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>418</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel" >
     <x>479</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbBottom</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ViewMPEG2</receiver>
   <slot>sbValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>431</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel" >
     <x>478</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbRight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ViewMPEG2</receiver>
   <slot>sbValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>411</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel" >
     <x>480</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>bCropDetectClick()</slot>
  <slot>sbValueChanged(int)</slot>
 </slots>
</ui>
